docs/token-contracts/fa2/2-fa2-nft-tutorial.md

```diff
@@ -0,0 +1,533 @@
+
+querying NFT contract KT1XP3RE6S9t44fKR9Uo5rAfqHvHXu9Cy7fh using balance inspector KT1Pezr7JjgmrPcPhpkbkH1ytG7saMZ34sfd
+requested NFT balances:
+owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 0	balance: 1
+owner: tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb	token: 1	balance: 0
+```
+
+Token `1` now belongs to `bob`.
+
+### Configuration
+
+`tznft` can be configured to interact with different Tezos networks. The user can
+also configure address aliases to sign Tezos operations and/or use them as command
+parameters when addresses are required. The default configuration that is created
+by `tznft init-config` command includes two pre-configured networks: `sandbox`
+and `testnet` (Carthagenet). Each pre-configured network has two bootstrap aliases:
+`bob` and `alice`.
+
+#### Network Configuration Commands
+
+- `set-network <network>` select specified pre-configured network as an active one.
+  All subsequent commands will operate on the active network
+
+  Example:
+
+  ```sh
+  $ tznft set-network sandbox
+
+  network sandbox is selected
+  ```
+
+- `show-network [--all]` show currently selected network. If `--all` flag is
+  specified, show all pre-configured networks
+
+  Example:
+
+  ```sh
+  $ tznft show-network --all
+
+  * sandbox
+    testnet
+  ```
+
+- `bootstrap` bootstrap selected network and deploy helper balance inspector contract.
+  If selected network is `sandbox` this command needs to be run each time sandbox
+  is restarted, for other public networks like `testnet` is it enough to run this
+  command once.
+
+  Example:
+
+  ```sh
+  $ tznft bootstrap
+
+  366b9f3ead158a086e8c397d542b2a2f81111a119f3bd6ddbf36574b325f1f03
+
+  starting sandbox...
+  sandbox started
+  originating balance inspector contract...
+  originated balance inspector KT1WDqPuRFMm2HwDRBotGmnWdkWm1WyG4TYE
+  ```
+
+- `kill-sandbox` stop Flextesa sandbox process if selected network is `sandbox`.
+  This command has no effect on other network types.
+
+  Example:
+
+  ```sh
+  $ tznft kill-sandbox
+
+  flextesa-sandbox
+
+  killed sandbox.
+  ```
+
+The sandbox network (selected by default) is configured to bake new Tezos blocks
+every 5 seconds. It makes running the commands that interact with the network
+faster. However, all originated contracts will be lost after the sandbox is stopped.
+
+If you are using `testnet`, your originated contracts will remain on the blockchain
+and you can inspect them afterwards using an block explorer like [BCD](https://better-call.dev/).
+
+_Note: Although `testnet` configuration already has two bootstrap aliases `bob`
+and `alice`, it is a good practice to create your own alias from the faucet file
+(see `tznft add-alias-faucet` command described below) and use it as a signer for
+the commands like `mint`, `transfer` and `show_balance`. In this way, your Tezos
+operations will not interfere with the operations initiated by other users._
```

Comment:
For example, you're more likely to get a "bad counter" error since someone else could use the counter for this block